dist/profile/manifests/archives.pp

```diff
@@ -238,10 +238,14 @@
     mode    => '0755',
   }
 
+  package { 'cron':
+    ensure => installed,
+  }
+
   cron { 'mirrorsync':
     command => '/usr/bin/mirrorsync',
     user    => 'mirrorsync',
     minute  => 30,
-    require => File['/usr/bin/mirrorsync'],
+    require => [File['/usr/bin/mirrorsync'],,Package['cron']],
```

Comment:
```suggestion
    require => [File['/usr/bin/mirrorsync'],Package['cron']],
```
If possible, add a space (?):
```suggestion
    require => [File['/usr/bin/mirrorsync'], Package['cron']],
```

Reply:
the PDK Puppet formatter removes the space :(

Thanks for the suggestion, I also saw the typo through the unit tests